docs/content/preview/explore/multi-region-deployments/asynchronous-replication-ysql.md

```diff
@@ -245,71 +199,61 @@ When the bidirectional replication has been configured, you can add data to the
 
 1. Use the following commands to add data to the "Data Center - West" cluster, making sure you are pointing to the new source host:
 
-   ```sh
-   ./bin/ysqlsh -h 127.0.0.2
-   ```
+    ```sh
+    ./bin/ysqlsh -h 127.0.0.2
+    ```
 
-     ```sql
-   INSERT INTO users(email, username) VALUES ('neha@example.com', 'neha'), ('mikhail@example.com', 'mikhail');
-     ```
+    ```sql
+    INSERT INTO users(email, username) VALUES ('neha@example.com', 'neha'), ('mikhail@example.com', 'mikhail');
+    ```
 
 2. On the new target cluster, run the following commands to see that data has been replicated between clusters:
 
-   ```sh
-   ./bin/ysqlsh -h 127.0.0.1
-   ```
+    ```sh
+    ./bin/ysqlsh -h 127.0.0.1
+    ```
 
-     ```sql
-   SELECT * FROM users;
+    ```sql
+    SELECT * FROM users;
      ```
 
-     You should see the following output:
+    You should see the following output:
 
-     ```output
-          email         | username
-   ---------------------+----------
-    hector@example.com  | hector
-    steve@example.com   | steve
-    neha@example.com    | neha
-    mikhail@example.com | mikhail
-   (4 rows)
-     ```
+    ```output
+            email         | username
+    ---------------------+----------
+      hector@example.com  | hector
+      steve@example.com   | steve
+      neha@example.com    | neha
+      mikhail@example.com | mikhail
+    (4 rows)
+    ```
 
 ## Add tables
 
-You can add more tables to an existing replication using the `yb-admin` command `alter_universe_replication` `add_table`:
+You can add more tables to an existing replication using the yb-admin command `alter_universe_replication` `add_table`:
 
-```sh
-yb-admin -master_addresses <target-master-addresses> alter_universe_replication <source-universe_uuid> add_table <source-table-ids>
+```sh.output
+yb-admin -master_addresses <target-master-addresses> \
+        alter_universe_replication <source-universe_uuid> \
+        add_table <source-table-ids>
 ```
 
 The following is an example command:
 
 ```sh
 ./bin/yb-admin -master_addresses 127.0.0.2:7100 \
-alter_universe_replication 7acd6399-657d-42dc-a90a-646869898c2d add_table 000030a9000030008000000000004000
+    alter_universe_replication 7acd6399-657d-42dc-a90a-646869898c2d \
+    add_table 000030a9000030008000000000004000
 ```
 
 For details, see [alter_universe_replication](../../../admin/yb-admin/#alter-universe-replication).
 
 ## Clean up
 
-
-You can choose to destroy and remove the clusters along with their associated directories.
-
-To stop the simulated data centers, you execute the `yugabyted stop` commands using the `--base_dir` option to specify the cluster.
-
-You can stop "Data Center - East" as follows:
+To destroy the simulated data centers and remove their associated directories, use the `yugabyted destroy` command as follows:
 
 ```sh
-./bin/yugabyted stop --base_dir=datacenter-east
+./bin/yugabyted destroy --base_dir=/tmp/ybd1
+./bin/yugabyted destroy --base_dir=/tmp/ybd2
```

Comment:
You can link from the sync replication page as a next step to this page as this is doing async replication. Or in general other pages from the multi-region section.